src/components/Courses/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin: 0 auto;
+  ${({ theme }) => theme.mq.md} {
+    width: 90%;
+  } ;
+`;
+const CoursesHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin: 0 auto;
+
+  button {
```

Comment:
To też oddzielnie, generalnie chyba wszystko lepiej jako oddzielny komponent pisać wszystkie h2, buttony bo potem jak coś to łatwiej to przerzucić do oddzielnego pliku i reużywalne zrobić